notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.        def _replace_saved_tensors(self, forward_trace, new_output_map):
</pre><p>wouldn't <a href="https://github.com/Lightning-AI/lightning-thunder/blob/815bff00f18825b9551e6dcf7b91f1f3c0dfb37b/thunder/core/symbol.py#L420" target="_blank">from_bsym_swap_proxies</a> be able to do the equivalent task in this method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>